rpcs3/main.cpp

```diff
@@ -473,7 +473,7 @@ int main(int argc, char** argv)
 	WSAStartup(MAKEWORD(2, 2), &wsa_data);
 #endif
 
-	ensure(thread_ctrl::is_main(), "Not main thread");
+	ensure(thread_ctrl::is_main(), FN(x), "Not main thread");
```

Comment:
i have to be honest. I have no idea what this does.

Reply:
Nothing, i.e. it takes first argument and returns it.

Reply:
but why is it needed

Reply:
you have an overload if i'm not mistaken.
So it seems kinda useless

Reply:
Right, it can be solved with overload resolution.